examples/tests/cypress/support/commands.js

```diff
@@ -48,7 +48,15 @@ Cypress.Commands.add('sendEvent', (eventType, message) => {
 
 Cypress.Commands.add('assertReceivedEvents', (element, message) => {
   // assert number and content of received events
-  cy.get(`#${element}`)
+  cy.get(`#${element} > li`, {timeout: 30000})
+    .should('be.visible')
+    .first()
+    .contains(message)
+});
+
+Cypress.Commands.add('assertReceivedEventLog', (element, message) => {
```

Comment:
I feel that it doesn't make sense to have two functions that are named similarly and almost do the same thing. Please find an API that better fits the has-the-event-been-received use case; passing the `element` is likely not part of it, btw.